sql/05-load-table-friend.sql

```diff
@@ -0,0 +1,20 @@
+INSERT INTO student.friend values (1510, 1381);
```

Comment:
indentation needs to be corrected here, and below, it is still just as readable but not in from a standpoint of syntactic adherence to form.